build/setup.mk

```diff
@@ -1,14 +1,16 @@
+GO_BUILD_FLAGS ?=
+
 # Ensure that go is installed. Note that this is independent of whether or not a server is being
 # built, since the build script itself uses go.
 ifeq ($(GO),)
     $(error "go is not available: see https://golang.org/doc/install")
 endif
 
 # Ensure that the build tools are compiled. Go's caching makes this quick.
-$(shell cd build/manifest && $(GO) build -o ../bin/manifest)
+$(shell cd build/manifest && $(GO) build -buildvcs=false $(GO_BUILD_FLAGS) -o ../bin/manifest)
 
 # Ensure that the deployment tools are compiled. Go's caching makes this quick.
-$(shell cd build/pluginctl && $(GO) build -o ../bin/pluginctl)
+$(shell cd build/pluginctl && $(GO) build -buildvcs=false $(GO_BUILD_FLAGS) -o ../bin/pluginctl)
```

Comment:
These changes may be unnecessary now. I think this is left over from when I was trying to build the plugin inside of a container https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container#example-running-a-job-within-a-container, instead of the Action runner process

Reply:
could you please check&remove them if they aren't required?